app/services/user-summary.js

```diff
@@ -0,0 +1,48 @@
+import Ember from 'ember';
+
+const { get, set } = Ember;
+
+export default Ember.Service.extend({
+  store: Ember.inject.service(),
+  authentication: Ember.inject.service(),
+
+  weeekEntries: [],
+  monthEntries: [],
+  
+  currentUserId: Ember.computed.reads('authentication.userId'),
+  
+  currentUserIdChanged: Ember.observer('currentUserId', function() {
+    this.reload();
+  }),  
```

Comment:
I would have kept this into the component instead of the service to prevent `reload` the service while its data are not displayed